docs/docsite/rst/administration/containers_instance_groups.rst

```diff
@@ -370,7 +385,7 @@ Container groups and instance groups are labeled accordingly.
 
 .. note::
 
-	Despite the fact that customers have custom Pod specs, upgrades may be difficult if the default ``pod_spec`` changes. Most any manifest can be applied to any namespace, with the namespace specified separately, most likely you will only need to override the namespace. Similarly, pinning a default image for different releases of the platform to different versions of the default job runner container is tricky. If the default image is specified in the Pod spec, then upgrades do not pick up the new default changes are made to the default Pod spec.
+	Despite the fact that customers have custom pod specs, upgrades may be difficult if the default ``pod_spec`` changes. Most any manifest can be applied to any namespace, with the namespace specified separately, most likely you will only need to override the namespace. Similarly, pinning a default image for different releases of the platform to different versions of the default job runner container is tricky. If the default image is specified in the pod spec, then upgrades do not pick up the new default changes are made to the default pod spec.
```

Comment:
"Most any manifest can be applied to any namespace, with the namespace specified separately, most likely you will only need to override the namespace."

Should this say "Since any"?

Reply:
"Despite the fact that customers have custom pod specs, upgrades may be difficult if the default ``pod_spec`` changes"

maybe,

"Note that using custom pod specs may cause issues on upgrade if the default pod spec changes"